public/javascripts/controllers/farmer-ctrl.js

```diff
@@ -105,6 +105,24 @@ angular.module('jasmic.controllers')
                 }
             };
 
+            $scope.isActive = function(exp_date) {
+                if(new Date(exp_date) > Date.now()) {
+                    return true;
+                } else {
+                    return false;
+                }
+            };
+
+            $scope.getMomentString = function(date_string) {
```

Comment:
This function is unnecessary and you should probably use angular's date filter in the HTML:
```
{{date_value | date}}
```